h/static/scripts/directives/thread.coffee

```diff
@@ -137,6 +142,12 @@ thread = [
         else
           attrs.$removeClass COLLAPSED_CLASS
 
+      scope.$watch $parse(attrs.thread), (thread) ->
```

Comment:
@tilgovi thanks, I've added a brief comment to the directive. https://github.com/hypothesis/h/commit/1a189f0985c6f893e310ad235266a975ce59f732